### PR TITLE
Feature/ui enable tracker analytics service

### PR DESCRIPTION
**Pull request by astewes**
-  Starts AuditMetrics service when Tracker is enabled

### CODE REVIEW

**Comment by elfenheart** [comment]
There is some problem with this when the AuditLog and AuditFlow programs are started but AuditMetrics is not, the enable tracker has to be clicked twice. Enabling Tracker on new namespace does not seem to work either..

**Comment by elfenheart** [comment]
I asked @astewes to change the numPartitions CDAP config key name because that key was deprecated in https://issues.cask.co/browse/CDAP-5264

**Comment by elfenheart** [comment]
@astewes let's close this PR and reopen against develop

**Comment by astewes** [comment]
@ajainarayanan per @elfenheart 's comment, this alright with you?

**Comment by ajainarayanan** [comment]
@astewes Yes. Lets not have dependent PRs. Lets close this and open it against develop.. We can merge [this PR](https://github.com/caskdata/cdap/pull/5939) & the new PR that you are opening against develop separately.. We add the check there to merge #5939 first before the new PR that you are about to create but that way we don't have to test the functionality that is merged to develop twice.

File: cdap-common/bin/service [hunk 1]
```diff
@@ -118,7 +118,7 @@ _start_java() {
   ulimit -a >>${loglog}
   echo "CLASSPATH=${CLASSPATH}" >>${loglog}
   # Start our JVM
-  local __defines="-Dcdap.service=${APP} ${JAVA_HEAPMAX} -Duser.dir=${LOCAL_DIR}"
+  local __defines="-Dcdap.service=${APP} ${JAVA_HEAPMAX} -Duser.dir=${LOCAL_DIR} -Djava.io.tmpdir=${TEMP_DIR}"
```
Comment (nitinmotgi) [inline cdap-common/bin/service:121]: Are we going to be cleaning the temp directory if there are things that keep pilling up.

**Pull request closed without merging**